Source/WebCore/Modules/permissions/MainThreadPermissionObserver.h

```diff
@@ -38,6 +38,7 @@
 namespace WebCore {
 
 class Page;
+class RegistrableDomain;
```

Comment:
Any reason not to use Site like the specification does? We should really stop using RegistrableDomain for most things.

Reply:
The on-disk database that tracks granted storage access uses RegistrableDomain, so I’m matching that here. Since `permissions.query` directly references that database, I don’t think we can use Site in this change without migrating the database first.